oshc/oshc/settings.py

```diff
@@ -21,7 +21,7 @@
 SECRET_KEY = 'x1-pogt0-b5owbgq0=ui02-4v)bba!bg&1m8_$)8-&13(907qf'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.getenv("DEBUG", False)
+DEBUG = os.getenv("DEBUG", True)
```

Comment:
Extraneous change. Please do not modify the files which are not related to the issue.